pkg/operator/starter.go

```diff
@@ -209,16 +210,22 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 			"v3.11.0/openshift-apiserver/svc.yaml",
 			"v3.11.0/openshift-apiserver/sa.yaml",
 			"v3.11.0/openshift-apiserver/trusted_ca_cm.yaml",
+			"v3.11.0/openshift-apiserver/audit-policies-cm.yaml",
 		},
 		kubeInformersForNamespaces,
 		kubeClient,
 	).WithRevisionController(
 		operatorclient.TargetNamespace,
 		nil,
-		[]revision.RevisionResource{{
-			Name:     "encryption-config",
-			Optional: true,
-		}},
+		[]revision.RevisionResource{
+			{
+				Name:     "encryption-config",
+				Optional: true,
+			},
+			{
+				Name: "openshift-apiserver-audit-policies",
```

Comment:
since this is coming from bindata I don't see how it can be revisioned

Reply:
I am using the static resource controller that creates the ConfigMap from the bindata first. that should work, right?